src/main.js

```diff
@@ -0,0 +1,4 @@
+const bookshelf = require('../database/connection.js')
```

Comment:
Yeah, I will delete it from this commit.

It's in a later PR this file is used to initialize the repl session.